oasislmf/execution/bash.py

```diff
@@ -1476,6 +1496,15 @@ def bash_params(
     else:
         bash_params['_get_getmodel_cmd'] = _get_getmodel_cmd
 
+    # Set custom gulcalc log statment checks, 
+    if custom_gulcalc_log_start and custom_gulcalc_log_finish:
+        bash_params['custom_gulcalc_log_start'] = custom_gulcalc_log_start
+        bash_params['custom_gulcalc_log_finish'] = custom_gulcalc_log_finish
+    else:
+        bash_params['custom_gulcalc_log_start'] = analysis_settings.get('model_custom_gulcalc_log_start', False)
+        bash_params['custom_gulcalc_log_finish'] = analysis_settings.get('model_custom_gulcalc_log_finish', False)
```

Comment:
The condition could be replaced with a simple `or`. This is done elsewhere. Although I admit I don't understand why the setting sometimes comes from the function parameters sometimes from the settings.

```suggestion
        bash_params['custom_gulcalc_log_start'] = custom_gulcalc_log_start or analysis_settings.get('model_custom_gulcalc_log_start')
        bash_params['custom_gulcalc_log_finish'] = custom_gulcalc_log_finish or analysis_settings.get('model_custom_gulcalc_log_finish')
```